.circleci/config.yml

```diff
@@ -1,51 +1,79 @@
 version: 2
+
+workflows:
+  version: 2
+  build_and_test:
+    jobs:
+      - build-builder
+      - build:
+          requires:
+            - build-builder
+
 jobs:
-  build:
+  build-builder:
     docker:
       - image: circleci/golang:1-stretch
     working_directory: /go/src/github.com/livepeer/go-livepeer
 
     environment:
-        PKG_CONFIG_PATH: $HOME/compiled/lib/pkgconfig
-        TEST_RESULTS: /tmp/test-results
+      PKG_CONFIG_PATH: /root/compiled/lib/pkgconfig
+      GOPATH: /go
 
     steps:
       - checkout
+      - setup_remote_docker:
+          # This actually only exists for paid plans. But it's good luck!
+          docker_layer_caching: true
+
+      - run: docker login -u $DOCKER_USER -p $DOCKER_PASS
+
+      # First, build the Linux-specific builder container
+      - run: docker pull iameli/build-platform:latest-linux || echo 'no pre-existing cache found'
```

Comment:
Just for sake of continuity, should this be held somewhere Livepeer-owned?

Reply:
I don't have access to the Docker Hub `livepeer` namespace — @ericxtang and I can transfer it next Wednesday (probs a realistic timetable for getting all the loose ends on this tied off anyway)